<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="GlView" name="glView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Eixo X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_Eixo_X">
          <property name="minimum">
           <double>-360.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Eixo Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_Eixo_Y">
          <property name="minimum">
           <double>-360.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Eixo Z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_Eixo_Z">
          <property name="minimum">
           <double>-360.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen3DModel"/>
    <addaction name="actionClose3DModel"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Report"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSerial_Port"/>
    <addaction name="separator"/>
    <addaction name="actionCamera"/>
    <addaction name="actionLight"/>
    <addaction name="actionMaterial"/>
    <addaction name="separator"/>
    <addaction name="actionTransformations"/>
    <addaction name="separator"/>
    <addaction name="actionUse_Radians"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuConfigura_es"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen3DModel">
   <property name="text">
    <string>Open 3D Model</string>
   </property>
  </action>
  <action name="actionClose3DModel">
   <property name="text">
    <string>Close 3D Model</string>
   </property>
  </action>
  <action name="actionSerial_Port">
   <property name="text">
    <string>Serial Port</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionMaterial">
   <property name="text">
    <string>Material</string>
   </property>
  </action>
  <action name="actionUse_Radians">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Radians</string>
   </property>
  </action>
  <action name="actionSave_Report">
   <property name="text">
    <string>Save Report</string>
   </property>
  </action>
  <action name="actionTransformations">
   <property name="text">
    <string>Transformations</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GlView</class>
   <extends>QWidget</extends>
   <header>glview.h</header>
   <container>1</container>
   <slots>
    <slot>rotateRelX(double)</slot>
    <slot>rotateAbsX(double)</slot>
    <slot>rotateX(double)</slot>
    <slot>rotateY(double)</slot>
    <slot>rotateZ(double)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
